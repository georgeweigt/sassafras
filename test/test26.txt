* Whatever ;
data ;
input y ;
yy = y**2 ;
datalines ;
1
2
3
4
5
6
;
proc print ;
