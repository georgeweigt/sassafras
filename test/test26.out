                             Obs     Y          YY
                               1     1      1.0000
                               2     2      4.0000
                               3     3      9.0000
                               4     4     16.0000
                               5     5     25.0000
                               6     6     36.0000

